Example/Shared/CoralogixRumManager.swift

```
@@ -25,13 +26,16 @@ final class CoralogixRumManager {
                                       userName: "?",
                                       userEmail: "a@a.com",
                                       userMetadata: ["d":"d"])
+        guard let publicKey = ProcessInfo.processInfo.environment["PUBLIC_KEY"] else {
+            fatalError("🚫 PUBLIC_KEY environment variable is not set.")
```

whats that? if a user didnt pass public key a debug/warn is enough
whats fatalError?

---

fatalError(_:) is a Swift function that terminates the program immediately at runtime.
It’s commonly used for:
Enforcing preconditions
Marking unimplemented functionality
Failing fast in invalid states